src/state/selectors.ts

```diff
@@ -48,6 +48,22 @@ export const getBuddyName = (
   );
 };
 
+export const getBuddyStatus = (
+  buddyId: string,
+  buddyState: AppState['buddies'],
+) => {
+  const look = (fa: RD.RemoteData<unknown, Record<string, { status: 'Banned' | 'Active' }>>) =>
+    RD.remoteData.map(fa, a => record.lookup(buddyId, a));
+  const buddy = getOptionM(RD.remoteData).alt(look(buddyState), () =>
+    look(buddyState),
+  );
+  return getFoldableComposition(RD.remoteData, O.option).reduce(
+    buddy,
+    '',
+    (_, { status }) => status,
+  );
+};
```

Comment:
This is not a selector by def. Selectors take whole AppState as their argument : ).

`const getbuddyStatus = buddyId => AppState => { ... }`

Reply:
And then in React side:

`const SomeCompoenent = (...) => {
...
useSeletor(isBuddyBanned(buddyId));

return (
  <SomeJsx>.
  ..
  </SomeJsx>
)
...
}`